### ISSUE TITLE
Keep 'loading' state when manual page reloading after login

### ISSUE THREAD

**Issue by shigeyf** [auth0-samples/auth0-react-samples#195]
Hi team,

This sample react app with react-auth0-spa.js does not work with manual page realoding after login.
When doing manual page reloading, the current login session does not seem to work and 'loading' spinner is show infinitely.
Also I saw that HTTP request to auth0.com/authorze returned 400 HTTP error after manual reloading the app page after login.

How can you fix the issue?

**Comment by stevehobbsdev** [comment]
@shigeyf Which browser are you seeing this in? Please can you also check your [Auth0 tenant logs](https://manage.auth0.com/#/logs) to see if any errors appear in there.

This type of thing is usually down to a configuration error.

**Comment by shigeyf** [comment]
Thanks for your reply, @stevehobbsdev 
I have seen this behavor (issue) on Chrome and Edge Chromium.
I checked Auth0 tenant logs and found the two same errors at almost same timing.

Occurred: 2 minutes ago at 2020-05-07 15:01:31.501 UTC
Occurred: 2 minutes ago at 2020-05-07 15:01:31.504 UTC
Type: Failed Silent Auth
Description: The specified redirect_uri 'http://localhost:3000' does not have a registered origin.

According to the error message, I checked again my application configurations. The configuration of "Allowed Web Origins" was "http://localhost:3000/".
I changed my original config ("http://localhost:3000/") to "http://localhost:3000" (without the last '/' character), and then it worked fine. 

It seems that the SPA SDK requires exact string match.
